docs/getting_started/installation.md

```diff
@@ -0,0 +1,55 @@
+# Installation
+
+## Pypi
+
+The module is available in pypi as
+[oras](https://pypi.org/project/oras/), and you can install as follows:
+
+``` console
+$ pip install oras
+```
+
+You can also clone the repository and install locally:
+
+``` console
+$ git clone https://github.com/oras-project/oras-py
+$ cd oras-py
+$ pip install .
+```
+
+Note that we have several extra modes for installation:
+
+``` console
+# interactions are done via the docker client instead of manual
+$ pip install oras[docker]
+
+# Install dependencies for linting and tests
+$ pip install oras[tests]
+
+# install everything
```

Comment:
For consistency, all sentences should be capitalized.

Reply:
All set!